packages/invoices/src/OutboundShipment/Dashboard/Widget.tsx

```diff
@@ -35,6 +35,7 @@ export const OutboundShipmentWidget: React.FC = () => {
   return (
     <>
       <NameSearchModal
+        type="customer"
```

Comment:
needed a way to differentiate search for suppliers or customers. I *think* this is OK.

The modal will likely end up being similar to https://github.com/sussol/msupply/blob/master/Project/Sources/Methods/nameFilterForListbox.4dm

so I've added some filtering things to the api changes in the spreadsheet - i.e. to do with special names and needing to filter out patient types